UnitTest/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "UnitTest": {
       "commandName": "Project",
-      "nativeDebugging": false
+      "nativeDebugging": true

```

Comment:
not intended for checkin?

Reply:
I feel like every time I debug C#/WinRT tests I need to turn this on. I can remove this if you'd like though.

Reply:
funny - me too.  I once checked it in (inadvertently) and you commented on that.  I'm fine with it.